src/main/inAppNotifications/inAppNotificationsDocs.md

```
@@ -0,0 +1,16 @@
+### In-App Notifications Documentation
+
+##### Add new Notification
```

Markdownlint (0.35.0)

Expected: h4; Actual: h5
Heading levels should only increment by one level at a time

(MD001, heading-increment)